default/python/AI/AIDependencies.py

```diff
@@ -143,11 +143,11 @@
 BASE_PLANET_STEALTH = 5
 
 STEALTH_STRENGTHS_BY_SPECIES_TAG = {
-    "BAD_STEALTH": -PRIMARY_FOCS_STEALTH_LEVELS["LOW_STEALTH"],
-    "GOOD_STEALTH": PRIMARY_FOCS_STEALTH_LEVELS["LOW_STEALTH"],
-    "GREAT_STEALTH": PRIMARY_FOCS_STEALTH_LEVELS["MEDIUM_STEALTH"],
-    "ULTIMATE_STEALTH": PRIMARY_FOCS_STEALTH_LEVELS["HIGH_STEALTH"],
-    "INFINITE_STEALTH": 500,  # used by super testers
+    "BAD": -PRIMARY_FOCS_STEALTH_LEVELS["LOW_STEALTH"],
```

Comment:
Does this speedup AI? :)

Reply:
This is a change to bring the dict into the form expected by the cached function get_species_tag_grade. So, indirectly, yes.